contrib/babelfishpg_tds/src/backend/tds/tdslogin.c

```diff
@@ -1769,15 +1776,15 @@ TdsClientAuthentication(Port *port)
 				char		hostinfo[NI_MAXHOST];
 
 				pg_getnameinfo_all(&port->raddr.addr, port->raddr.salen,
-								   hostinfo, sizeof(hostinfo),
```

Comment:
Unnecessary changes. Please discard the same, otherwise the git blame becomes difficult.

Reply:
Done